Demos/Shopping/ShoppingUITests/TestPages/CartPage.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  CartPage.swift
+//  ShoppingUITests
+//
+//  Created by Albert Bori on 1/6/23.
+//
+
+import XCTest
+
+/// The test view for the cart which provides the ability to remove cart items and checkout
+struct CartPage<ParentView: TestableUI>: PresentedPage {
+    let app: XCUIApplication
+    var parentView: ParentView
+    
```

Comment:
I would suggest defining all elements in a page at the top of a class probably using `var` or `lazy` so that you can 1 see what is in a page at a glance and 2 so that if the UI changes like an id or trait are changed you only have to update a single line of code in your tests

Reply:
Great suggestion! I did just as you recommended. Please review the latest push to this PR when you have a moment.